CONTRIBUTING.md

```diff
@@ -331,93 +330,6 @@ satisfy the tests provided for each problem.
 You can include advice and helpful links for your language track in the
 `SETUP.md` file.
 
-## Maintaining a Track
```

Comment:
Contains information not covered in https://github.com/exercism/docs/tree/master/maintaining-a-track